build-scripts/rollup.browser.test.config.js

```diff
@@ -18,7 +19,7 @@ export default function createBrowserTestConfig(options = {}) {
           `,
           outro: '});'
         },
-        plugins: [multiEntry()]
+        plugins: [alias({ '@pollyjs-tests': testsPath }), multiEntry()]
```

Comment:
Add an alias to our common `tests` directory so we dont have to use relative paths everywhere.